<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindSubtitlesConfigDialog</class>
 <widget class="QDialog" name="FindSubtitlesConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="misc_group">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Search &amp;method:</string>
          </property>
          <property name="buddy">
           <cstring>search_method_combo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="search_method_combo"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="addlang_check">
        <property name="text">
         <string>A&amp;ppend language code to the subtitle filename</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="credentials_group">
     <property name="title">
      <string>Opensubtitles Credentials</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Username:</string>
          </property>
          <property name="buddy">
           <cstring>os_username_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="os_username_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Pa&amp;ssword:</string>
          </property>
          <property name="buddy">
           <cstring>os_password_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="os_password_edit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxy_group">
     <property name="title">
      <string>Proxy</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QCheckBox" name="use_proxy_check">
        <property name="text">
         <string>&amp;Enable proxy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="proxy_container" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&amp;Host:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_hostname_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLineEdit" name="proxy_hostname_edit"/>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;Port:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="buddy">
            <cstring>proxy_port_spin</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QSpinBox" name="proxy_port_spin">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;Username:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_username_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QLineEdit" name="proxy_username_edit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Pa&amp;ssword:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_password_edit</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QLineEdit" name="proxy_password_edit">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&amp;Type:</string>
           </property>
           <property name="buddy">
            <cstring>proxy_type_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="proxy_type_combo"/>
         </item>
         <item row="3" column="2" colspan="3">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>261</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="server_group">
     <property name="title">
      <string>Server</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="server_label">
          <property name="text">
           <string>&amp;OpenSubtitles server:</string>
          </property>
          <property name="buddy">
           <cstring>server_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="server_edit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="retries_label">
          <property name="text">
           <string>Number of &amp;retries:</string>
          </property>
          <property name="buddy">
           <cstring>retries_spin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="retries_spin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>257</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>search_method_combo</tabstop>
  <tabstop>addlang_check</tabstop>
  <tabstop>os_username_edit</tabstop>
  <tabstop>os_password_edit</tabstop>
  <tabstop>use_proxy_check</tabstop>
  <tabstop>proxy_hostname_edit</tabstop>
  <tabstop>proxy_port_spin</tabstop>
  <tabstop>proxy_username_edit</tabstop>
  <tabstop>proxy_password_edit</tabstop>
  <tabstop>proxy_type_combo</tabstop>
  <tabstop>server_edit</tabstop>
  <tabstop>retries_spin</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FindSubtitlesConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FindSubtitlesConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_proxy_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>proxy_container</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
